pkg/webhook/conv.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	admissionreg "k8s.io/api/admissionregistration/v1"
+	admissionregv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	conversionScheme = runtime.NewScheme()
+)
+
+func init() {
+	if err := admissionreg.AddToScheme(conversionScheme); err != nil {
```

Comment:
More for my understanding, why did we not need this before? Also, if you panic its a lot nicer for everyone involved to include the error in the panic :)

Reply:
This is not needed before because we only support a single version of WebhookConfiguration, we just created the versioned `v1beta1` objects in the parser.go, and be done with it.
Now we have 2 versions, so we can let the scheme to handle conversion between them.

I will update the panic :) I didn't take look that closely, and copied that from crd package :)

Reply:
Done